packages/astro/e2e/errors.test.js

```diff
@@ -52,6 +52,28 @@ test.describe('Error display', () => {
 		expect(await page.locator('vite-error-overlay').count()).toEqual(0);
 	});
 
+	test('shows correct file path when a page has an error', async ({ page, astro }) => {
```

Comment:
Two different tests because on Windows the path separators are different depending on the location of the error (i.e. component vs page).